components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2010,11 +2035,15 @@ std::string AdsServiceImpl::LoadDataResourceAndDecompressIfNeeded(
 
 void AdsServiceImpl::ShowNotification(
     const std::unique_ptr<ads::AdNotificationInfo> info) {
-  auto notification = CreateAdNotification(*info);
-
-  display_service_->Display(NotificationHandler::Type::BRAVE_ADS,
-      *notification, /*metadata=*/nullptr);
+  LOG(INFO) << "albert AdsServiceImpl::ShowNotification";
+  brave_custom_notification::Notification* notification = (brave_custom_notification::Notification*) CreateAdNotification(*info);
 
+  // Call NotificationPlatformBridgeBraveCustomNotification
+  NotificationPlatformBridgeBraveCustomNotification* platform_bridge = new NotificationPlatformBridgeBraveCustomNotification(profile_);
+  platform_bridge->Display(NotificationHandler::Type::BRAVE_ADS, profile_, *notification, /*metadata=*/nullptr);
```

Comment:
Is there a reason we still need to pass BRAVE_ADS type as NotificationPlatformBridgeBraveCustomNotification only displays Brave Ads